app/assets/javascripts/oxalis/view/tracing_layout_view.js

```diff
@@ -77,7 +77,7 @@ class TracingLayoutView extends React.PureComponent {
             />
 
             <Layout className="tracing-layout">
-              <Header>
+              <Header style={{ position: "fixed", width: "100%", zIndex: 210, minHeight: 48 }}>
```

Comment:
Low-prio: Any possibility those numbers could be less magic? If the height of the header is defined in some css file, I would suggest to define the minHeight there as well and make them dependent on the same variable. Same can probably be done for that marginTop value.